### PR TITLE
status: 2023q1: clusteradm: more minor changes

### PR DESCRIPTION
**Pull request by grahamperrin**
@dbaio

### CODE REVIEW

File: website/content/en/status/report-2023-01-2023-03/clusteradm.adoc [hunk 1]
```diff
@@ -13,7 +13,7 @@ In this quarter, the team has worked on the following:
 * Regular disk and parts support (and replacement) for all physical hosts and mirrors.
 * Improve the PowerPC package builders.
 ** With new parts obtained through the FreeBSD Foundation, the builders now have new NVMEs with heatsinks and more memory.
-It helped seize the heat issue, and they are building packages faster now.
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-01-2023-03/clusteradm.adoc:16]: According to <https://ell.stackexchange.com/a/244800/31430>, the verb _seize_ is an old French legal term that means _to be put in legal possession of_. 

In modern English, the verb commonly used in phrases such as [seize stolen goods](https://www.startpage.com/do/dsearch?query=seize+stolen+goods&cat=web&language=english) … I would not use it in relation to a hardware issue.

File: website/content/en/status/report-2023-01-2023-03/clusteradm.adoc [hunk 2]
```diff
@@ -31,7 +31,7 @@ This task is in conjunction with the FreeBSD Foundation and donors/sponsors.
 
 ==== FreeBSD Official Mirrors Overview
 
-Current locations are Australia, Brazil, Germany, Japan (two full mirror sites), Malaysia, South Africa, Taiwan, United Kingdom (full mirror site), United States of America (California, New Jersey [the primary site], and Washington).
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-01-2023-03/clusteradm.adoc:34]: Square brackets within parentheses look very weird, to me. Maybe weird to my UK eye – until today, I assumed that square brackets were a US preference (so, it looked like a mixture of Americanised within de-Americanised). 

Today I found <https://english.stackexchange.com/a/11173/11504> –

>> Nesting parentheses should be done with square brackets "([ ])". …

– but still, it looks weird. 

----

Taking <https://english.stackexchange.com/q/11155/11504> and <https://english.stackexchange.com/q/553839/11504> as a whole, including comments, I lean towards _avoidance_ of square brackets (or parentheses) within parentheses.

**Comment by grahamperrin** [comment]
Whilst checking for broken URLs within the _generously provided by_ list, I found English alternatives for two of the Japanese pages: 

* <https://www.bbtower.co.jp/en/corporate/>
* <https://www.kddi-webcommunications.co.jp/english/>. 

Whilst the status report is in English, I think it's OK to link to the Japanese versions (it's easy enough to click through from Japanese to English).

File: website/content/en/status/report-2023-01-2023-03/clusteradm.adoc (hunk 1 above)
Comment (dbaio) [reply website/content/en/status/report-2023-01-2023-03/clusteradm.adoc:16]: Agreed.

File: website/content/en/status/report-2023-01-2023-03/clusteradm.adoc [hunk 3]
```diff
@@ -31,7 +31,7 @@ This task is in conjunction with the FreeBSD Foundation and donors/sponsors.
...
+Current locations are Australia, Brazil, Germany, Japan (two full mirror sites), Malaysia, South Africa, Taiwan, United Kingdom (full mirror site), United States of America -- California, New Jersey (primary site), and Washington.
```
Comment (dbaio) [inline website/content/en/status/report-2023-01-2023-03/clusteradm.adoc:34]: much better, thanks

File: website/content/en/status/report-2023-01-2023-03/clusteradm.adoc [hunk 4]
```diff
@@ -43,7 +43,7 @@ The hardware and network connection have been generously provided by:
 * https://www.isc.org/[Internet Systems Consortium]
 * https://www.inx.net.za/[INX-ZA]
 * https://www.kddi-webcommunications.co.jp/[KDDI Web Communications Inc]
-* https://myren.net.my/[Malaysian Research & Education Network]
+* https://www.mohe.gov.my/en/services/research/myren[Malaysian Research & Education Network]
```
Comment (dbaio) [inline website/content/en/status/report-2023-01-2023-03/clusteradm.adoc:46]: Also need to update the [internal/about](https://www.freebsd.org/internal/about/) page.
Could you take care of it, please? Along with the other URLs?

Then I will update the same in the mirrors:
http://pkg0.kul.freebsd.org/
http://ftp0.kul.freebsd.org/

**Review by dbaio — APPROVED** [review]

File: website/content/en/status/report-2023-01-2023-03/clusteradm.adoc (hunk 4 above)
Comment (grahamperrin) [reply website/content/en/status/report-2023-01-2023-03/clusteradm.adoc:46]: Thanks

> Also need to update the [internal/about](https://www.freebsd.org/internal/about/) page. Could you take care of it, please? Along with the other URLs?

▶ https://github.com/freebsd/freebsd-doc/pull/157

> Then I will update the same in the mirrors: [http://pkg0.kul.freebsd.org](http://pkg0.kul.freebsd.org/) [http://ftp0.kul.freebsd.org](http://ftp0.kul.freebsd.org/)

**Comment by dbaio** [comment]
Thanks.

**Comment by lsalvadore** [comment]
@grahamperrin: What is the status of this pull request? Has everything been merged or is there still something preventing it from being closed?

**Comment by grahamperrin** [comment]
Thanks for the nudge. 

I left it open to remind myself of Danilo's intention above <https://github.com/freebsd/freebsd-doc/pull/156#discussion_r1160712063> re: the two mirrors. 

<http://pkg0.kul.freebsd.org/> now does use a working URL for the _Malaysian Research & Education Network_. 

<http://ftp0.kul.freebsd.org/> does not, yet, so I ping @dbaio and close this :-)

**Comment by grahamperrin** [comment]
> … Has everything been merged …

Sorry, evidently not. 

I see the word _seize_ under <https://www.freebsd.org/status/report-2023-01-2023-03/#_cluster_administration_team>, contrary to <[`9edd6da` (#156)](https://github.com/freebsd/freebsd-doc/pull/156/commits/9edd6da12b4e8a2f24c31882dca86500b6803634)> … and so on. 

@lsalvadore in retrospect: I see, now, that I was awaiting your (`status`) approval. 

I can not reopen this PR because I deleted the branch. I'll open a new PR.

**Pull request closed without merging**